### PR TITLE
Add 2D plots for the comparison between data and emulator for L1TObjects - L1T&Online DQM - cmssw102X

### PR DESCRIPTION
**Pull request by pgianneios**
This PR applies the changes that have been discussed in https://its.cern.ch/jira/browse/CMSLITDPG-166

It introduces a new boolean parameter enable2DComp in:
DQM/L1TMonitor/python/L1TStage2uGMT_cff.py 
and
DQM/L1TMonitor/python/L1TdeStage2CaloLayer2_cfi.py

which can be set to True for drawing 2D eta-phi mismatch plots for the comparison of two collections of L1TOblects.

When enable2DComp=True in DQM/L1TMonitor/python/L1TStage2uGMT_cff.py then 2D plots will be produced for Muon Collections and when enable2DComp=True in DQM/L1TMonitor/python/L1TdeStage2CaloLayer2_cfi.py then 2D plots will be produced for Jet, EGs and Tau collections.

### CODE REVIEW

**Comment by thomreis** [comment]
please test

File: DQM/L1TMonitor/python/L1TdeStage2CaloLayer2_cfi.py [hunk 1]
```diff
@@ -10,5 +10,6 @@
     calol2TauCollectionEmul = cms.InputTag("valCaloStage2Layer2Digis"),
     calol2EtSumCollectionData = cms.InputTag("caloStage2Digis", "EtSum"),
     calol2EtSumCollectionEmul = cms.InputTag("valCaloStage2Layer2Digis"),
-    monitorDir = cms.untracked.string("L1TEMU/L1TStage2CaloLayer2/L1TdeStage2CaloLayer2")
+    monitorDir = cms.untracked.string("L1TEMU/L1TStage2CaloLayer2/L1TdeStage2CaloLayer2"),
+    enable2DComp = cms.untracked.bool(False) # When true eta-phi comparison plots are also produced
```
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TdeStage2CaloLayer2_cfi.py]: You can enable the 2D plots here.

File: DQM/L1TMonitor/python/L1TdeStage2uGMT_cff.py [hunk 2]
```diff
@@ -72,6 +72,7 @@
     muonCollection2Title = cms.untracked.string("uGMT emulator"),
     summaryTitle = cms.untracked.string("Summary of comparison between uGMT muons and uGMT emulator muons"),
     verbose = cms.untracked.bool(False),
+    enable2DComp = cms.untracked.bool(False), # When true eta-phi comparison plots are also produced
```
Comment (thomreis) [inline DQM/L1TMonitor/python/L1TdeStage2uGMT_cff.py]: You can enable the 2D plots here.

**Comment by thomreis** [comment]
please test

**Comment by fabiocos** [comment]
+1

**Pull request merged**